src/PKSim.Presentation/Core/MenuBarItemIds.cs

```diff
@@ -47,6 +47,9 @@ public static class MenuBarItemIds
       public static MenuBarItemId NewObserverSet = createMenuBarItemId("NewObserverSet");
       public static MenuBarItemId LoadObserverSet = createMenuBarItemId("LoadObserverSet");
       public static MenuBarItemId NewExpressionProfile = createMenuBarItemId("NewExpressionProfile");
+      public static MenuBarItemId NewExpressionProfileEnzyme = createMenuBarItemId("NewExpressionProfileEnzyme");
+      public static MenuBarItemId NewExpressionProfileTransporter = createMenuBarItemId("NewExpressionProfileTransporter");
+      public static MenuBarItemId NewExpressionProfileSpecificBindingPartner = createMenuBarItemId("NewExpressionProfileSpecificBindingPartner");
```

Comment:
This does then something like this

![image](https://user-images.githubusercontent.com/1041237/140412022-240f5130-ab1a-432c-bf63-667d88ee6eaa.png)


or like this

![image](https://user-images.githubusercontent.com/1041237/140412070-286951b6-795a-41b0-90c0-be5f90633bfd.png)